draft-ietf-quic-transport.md

```diff
@@ -4697,11 +4697,11 @@ active_connection_id_limit (0x000e):
   to store. This value includes the connection ID received during the handshake,
   that received in the preferred_address transport parameter, and those received
   in NEW_CONNECTION_ID frames.
-  Unless a zero-length connection ID is being used, the value of the
+  Unless the peer is using a zero-length connection ID, the value of the
```

Comment:
I would instead prefer to say that the value is assumed to be zero if the peer requests that a zero length CID be used.

Reply:
Rather than "assumed to be zero," just say that value of the TP is not used in that case.

Reply:
I've changed this to say that the TP is ignored if an endpoint is using a zero-len CID.